windows/UpdateManager.cs

```diff
@@ -0,0 +1,269 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.Web.Http;
+
+namespace CodePush.ReactNative
+{
+    internal class UpdateManager
+    {
+        // Internal methods
```

Comment:
If we want to achieve tooling support for grouping method definitions, you could use the [#region directive](https://msdn.microsoft.com/en-us/library/9a1ybwek.aspx), which allows expanding/collapsing the entire group. This is somewhat equivalent to our use of #pragma in Obj-C.